backend/climateconnect_main/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for climateconnect_main project.
+
+Generated by 'django-admin startproject' using Django 2.2.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+from dotenv import find_dotenv, load_dotenv
+load_dotenv(find_dotenv('.pyenv'))
+env = os.environ.get
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = env('SECRET_ENV')
+
+# SECURITY WARNING: don't run with debug turned on in production!
```

Comment:
Once this is merged, we should add an issue for this.

Reply:
Yes, I agree. cc/ @positiveimpact